webrtc.html

```diff
@@ -8748,13 +8758,13 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </div>
       <div>
         <pre class="idl">dictionary RTCDataChannelInit {
-             boolean        ordered = true;
-             unsigned short maxPacketLifeTime;
-             unsigned short maxRetransmits;
-             USVString      protocol = "";
-             boolean        negotiated = false;
+             boolean         ordered = true;
+             unsigned short  maxPacketLifeTime;
+             unsigned short  maxRetransmits;
+             USVString       protocol = "";
+             boolean         negotiated = false;
              [EnforceRange]
-             unsigned short id;
+             unsigned short  id;
```

Comment:
Decided to keep this white space change since it makes things line up with "priority"